lib/zendesk_apps_support/validations/translations.rb

```diff
@@ -13,13 +13,15 @@ class TranslationFormatError < StandardError
       end
 
       class << self
-        def call(package)
+        def call(package, opts = {})
           package.files.each_with_object([]) do |file, errors|
             path_match = TRANSLATIONS_PATH.match(file.relative_path)
             next unless path_match
             errors << locale_error(file, path_match[1]) << json_error(file) << format_error(file)
             next unless errors.compact.empty?
-            errors.push(*validate_marketplace_content(file, package)) if file.relative_path == 'translations/en.json'
+            if file.relative_path == 'translations/en.json'
+              errors.push(*validate_marketplace_content(file, package, opts[:skip_marketplace_translations]))
```

Comment:
i know you said nil is fine, but i still prefer `opts.fetch(:skip_marketplace_translations, false)` ;-)